clarifai/utils/logging.py

```diff
@@ -73,11 +73,21 @@ def _get_library_name() -> str:
   return __name__.split(".")[0]
 
 
-def _configure_logger(logger_level: str = "ERROR") -> None:
-  logging.basicConfig(
-      level=logger_level,
-      datefmt='%Y-%m-%d %H:%M:%S',
```

Comment:
How is the date format set now?

Reply:
Great catch. Added back the same format now.